tests/test_lab/test_gates_fock.py

```diff
@@ -317,3 +317,12 @@ def test_measure_with_fock():
     n_detect = 2
     state_out = state << Fock([n_detect], modes=[1])
     assert np.allclose(state_out.ket(), np.array([0.00757899, 0.0]))
+
+
+@given(theta=angle, phi=angle)
+def test_schwinger_bs_equals_vanilla_bs(theta, phi):
+    """Tests that the Schwinger boson BS gate is equivalent to the vanilla BS gate for low cutoffs."""
```

Comment:
We can remove this line with the more explicitely-named test

Reply:
sorry, which line?